cockatrice/src/tab_deck_editor.cpp

```diff
@@ -863,6 +873,43 @@ void TabDeckEditor::actOpenCustomsetsFolder() {
 
 }
 
+void TabDeckEditor::actAddCustomSet()
+{
+
+#if QT_VERSION < 0x050000
```

Comment:
Personal preference; Can you check against [`QT_VERSION_CHECK`](http://doc.qt.io/qt-5/qtglobal.html#QT_VERSION_CHECK) instead of the hex string? I find it more readable.